NAMESPACE

```diff
@@ -147,6 +148,7 @@ importFrom(lifecycle,deprecated)
 importFrom(magrittr,"%>%")
 importFrom(rlang,"%||%")
 importFrom(rlang,":=")
+importFrom(rlang,.data)
```

Comment:
```suggestion
```

This looks like cruft from before — I suspect `devtools::document()` would do this too

Reply:
Oh yes, I need to run it.